dll/win32/windowscodecs/CMakeLists.txt

```diff
@@ -80,6 +85,6 @@ endif()
 
 set_module_type(windowscodecs win32dll)
 target_link_libraries(windowscodecs wine uuid ${PSEH_LIB})
-add_importlibs(windowscodecs ole32 oleaut32 rpcrt4 shlwapi user32 gdi32 advapi32 advapi32_vista propsys msvcrt kernel32 ntdll)
+add_importlibs(windowscodecs png libjpeg tiff ole32 oleaut32 rpcrt4 shlwapi user32 gdi32 advapi32 advapi32_vista propsys kernel32_vista msvcrt kernel32 ntdll)
```

Comment:
In "png libjpeg tiff" , aren't these libs called "libpng", ..., "libtiff" ? (and so, you would have to use these names instead)